src/lib/language.ml

```diff
@@ -279,7 +303,7 @@ let rec to_shell: type a. _ -> a t -> string =
           [
             "while :;"; " do case $1 in";
             "-h|-help|--help) ";
-            sprintf "export %s_help=0 ; " prefix;
+            sprintf "export %s_help=true ; " prefix;
```

Comment:
I think in the case of `sprintf` have a long readable name that everybody knows makes things more readable (espcially in the case of functions that have "weird" typing like this one).